fcl/dunefd/reco/atmo_reco_1dsignalprocessing_dune10kt_1x2x6_refactored.fcl

```diff
@@ -0,0 +1,38 @@
+#include "reco_dune10kt_1x2x6_wirecell_refactored.fcl"
+
+process_name: rereco
+
+physics.producers:
+{
+    @table::physics.producers
+    caldata:            @local::producer_adcprep
+    wclsmcnfsp:         @erase
+}
```

Comment:
If the fcl inherits from reco_1dsignalprocessing_dune10kt_1x2x6_refactored.fcl, instead of reco_dune10kt_1x2x6_wirecell_refactored.fcl then I don't think any of these lines are needed.